source/logging/StdOutLogger.cpp

```diff
@@ -31,6 +31,7 @@ void StdOutLogger::run()
         {
             writeLogMessage(std::move(message));
         }
+        this_thread::sleep_for(std::chrono::milliseconds(25));
```

Comment:
How is the 25 determined? 
Have we verified the sleep time cause any impact on the logging ability of device client? For example, the logs writing has delay?

Reply:
I've tested this, logging functionality doesn't get impacted by this sleep time. If we set sleep time too low the cpu utilization will increase but overall I've tested 25ms works fine  my testing machine